rocketchat/templates/poddisruptionbudget.yaml

```diff
@@ -1,8 +1,9 @@
 {{- if and (.Values.podDisruptionBudget.enabled) (gt ( .Values.replicaCount | int ) 1) }}
-{{- if semverCompare ">=1.21.0" .Capabilities.KubeVersion.GitVersion }}
-apiVersion: policy/v1
-{{- else }}
+{{- $kubeversion := include "rocketchat.kubeversion" . }}
+{{- if semverCompare "<=1.21.0" $kubeversion }}
 apiVersion: policy/v1beta1
+{{- else }}
+apiVersion: policy/v1
```

Comment:
Do we really want all that extra helper stuff?  Or should we just upgrade the policy/v1 ?

Reply:
I don't know if it breaks other customers, but from my perspective i agree. 

Maybe this is an alternative solution with less complexity:
https://stackoverflow.com/questions/66147091/is-it-possible-to-configure-a-helm-chart-to-dynamically-use-a-value-based-on-ano
```
{{- if .Capabilities.APIVersions.Has "apps/v1" }}
apiVersion: apps/v1
{{- else }}
apiVersion: apps/v1beta1
{{- end }}
```

If you want i can change the pull request.

Reply:
policy/v1 has been there since 1.21 I see, makes sense to just get rid of the condition fluff.

don't know why i added those checks to begin with 🙈 

ref: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125